api/src/Listener/UserCreateListener.php

```diff
@@ -31,18 +46,34 @@ public function __construct(KernelInterface $kernel, UserPasswordEncoderInterfac
     public static function getSubscribedEvents(): array
     {
         return [
-            Events::prePersist,
+            Events::postPersist
         ];
     }
 
-    public function prePersist(LifecycleEventArgs $args): void
+    public function postPersist(LifecycleEventArgs $args): void
     {
         $user = $args->getObject();
 
         if (!$user instanceof User) {
             return;
         }
 
+        try {
+            $this->client->request(
+                self::HTTP_CODI_METHOD,
```

Comment:
```suggestion
                Request::METHOD_POST,
```

From the `HttpFoundation` namespace.